R/test_nse.R

```diff
@@ -17,7 +17,7 @@ test_fun <- function(select, data, exclude, ignore_case) {
   # in case pattern is a variable from another function call...
   p <- try(eval(select), silent = TRUE)
   if (inherits(p, c("try-error", "simpleError"))) {
-    p <- substitute(select)
+    p <- substitute(select, env = parent.frame())
```

Comment:
![what](https://user-images.githubusercontent.com/26301769/159073912-ba5562b9-d75a-4bf7-8913-c767e984a709.gif)

Reply:
Gotta tell R where to look!